storage_service/static/js/project.js

```diff
@@ -36,6 +37,12 @@ $(document).ready(function() {
   packagesDataTableOptions["bServerSide"] = true;
   packagesDataTableOptions["bProcessing"] = true;
   packagesDataTableOptions["sAjaxSource"] = uri + "packages_ajax";
+  if (location !== null && location !== "") {
```

Comment:
Would the following be a better option?

```js
if (typeof(location) === "string" && location.length) { ... }
```

Since `data` may return `undefined` (not `null`?) but we'd only care when it's a `string`.